config/jobs/kubernetes/test-infra/janitors.yaml

```
@@ -54,17 +54,21 @@ periodics:
   name: maintenance-pull-janitor
   labels:
     preset-service-account: "true"
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: k8s.io/test-infra
   spec:
     containers:
-    - args:
+    - command:
+      - runner.sh
```

this command doesn't do much on its own.
still need to run the "janitor" logic

---

```suggestion
      - runner.sh
      - "scenarios/kubernetes_janitor.py"
```

that is what I was thinking, but I wasn't sure. Would it just be as simple as passing in the current scenario script? like https://gist.github.com/dims/c1296f8ed42238baea0a5fcae45f4cf4#file-after-yaml-L36-L38 or should the scenario script be moved?

---

I think in the past we've used the scenario file as-is to start.
Sorry, my bandwidth will be very poor for the next few weeks with https://kubernetes.slack.com/archives/CCK68P2Q2/p1679420278418599 / https://kubernetes.io/blog/2023/03/10/image-registry-redirect/

---

updated! No worries, I know you are busy :)